docs/advanced-topics/docker-alternatives.rst

```diff
@@ -73,6 +73,36 @@ add a line to your ``~/.dockstore/config``:
 
     cromwell-vm-options: -Dconfig.file=<absolute path to your Cromwell conf>
 
+Podman
+---------------
+
+Podman is a daemon-less alternative of Docker that allows end users to run docker/OCI containers without root privileges.
+
+Podman can be installed following the instructions
+`here <https://podman.io/getting-started/installation>`__.
+
+cwltool
+~~~~~~~
+
+Podman is also available as a command line option for cwltool like this:
+::
+
+    cwltool --podman <workflow> <input json>
+
+To set this option through Dockstore, add the following line to your ``~/.dockstore/config``:
+::
+
+    cwltool-extra-parameters: --podman
+
+Cromwell
+~~~~~~~~
+For cromwell, we can set a symbolic link to run Podman instead of docker on workflows.
+::
+
+    ln -s /usr/bin/podman /usr/bin/docker
+
+Note that docker should not be installed when running this command.
+
 Rootless Docker
 ---------------
 
```

Comment:
Is the stuff in this section still true? Either way, it's probably best to combine this section with the one you edited around line 19.

Reply:
I tested the other alternatives like Singularity and they all work with Docker installed. Except for podman as the symbolic link causes errors when Docker is installed